src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -1,35 +1,38 @@
-import { useState } from 'react';
-import { BurgerMenuIcon, RightArrow } from '@/assets/constants';
-import { Button, Drawer } from 'antd';
+import { RightArrow } from '@/assets/constants';
+import { Drawer } from 'antd';
 import styles from './menu.module.scss';
 import { Link } from 'react-router-dom';
 import items from './constants';
 import ButtonNav from '../Button/Button';
 import buttonData from '@/constants/ButtonConstants';
+import { Header } from '../components';
+import { useMenuContext } from '@/context/menuContext.ts';
 
 export default function BurgerMenu() {
-  const [open, setOpen] = useState(false);
+  //const [open, setOpen] = useState(false);
+  const { onMenuClose, isMenuOpen } = useMenuContext();
 
-  const showDrawer = () => {
-    setOpen(true);
-  };
-
-  const closeDrawer = () => {
-    setOpen(false);
-  };
   return (
     <>
-      <Button onClick={showDrawer} type="text">
-        <img src={BurgerMenuIcon} />
-      </Button>
-
       <Drawer
         closable={false}
-        onClose={closeDrawer}
-        open={open}
+        width={'100vh'}
```

Comment:
`vh` its height (vertical), mayby you mean `vw`?
Please check the code, as I don't fully understand what it's supposed to do at first glance.